e2e/cypress/integration/news.spec.js

```diff
@@ -20,8 +28,66 @@ describe('News and Announcements', () => {
     cy.findByRole('link', { name: 'News' }).click()
     cy.url().should('eq', Cypress.config().baseUrl + '/news')
     cy.findByRole('heading', { level: 1 }).contains('News')
-    cy.wait(['@getNewsRSS'])
+    cy.wait('@getNewsRSS')
     cy.findAllByRole('article').should('exist')
     cy.checkA11y(null, null, logging, { skipFailures: true })
   })
+
+  it('can add the News Section to My Space', () => {
```

Comment:
Thinking out loud here, but from your PR body, you say this `Since right now there can only be one News widget instance, this errors if a user tries to add a second one ` is there value in testing this behavior?

Reply:
this is tested here on [L53](https://github.com/USSF-ORBIT/ussf-portal-client/pull/522/files/7bb3e8a13eb43c605cbcfc75eeb60e92e58ef230#diff-8211da167b81c900e444d27c7ee890dcacd331afd2aa43fdecddd348a9ef10ebR53) by asserting that the button becomes disabled. it's also unit tested in the My Space model test at the database level.